package/Shaders/Lighting.hlsl

```diff
@@ -1602,7 +1641,7 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 	float puddle = wetness;
 	if (wetness > 0.0 || puddleWetness > 0) {
 #		if !defined(SKINNED)
-		puddle = noise(puddleCoords) * ((minWetnessAngle / perPassWetnessEffects[0].PuddleMaxAngle) / 1.5) + 0.5;
```

Comment:
@doodlum I would like to keep this part since it helps keep puddles from forming on vertical surfaces like walls and mountain sides etc. If you are seeing a bug let me know how you run into it and I'll find a fix.

Reply:
@doodlum reminder about this request.

Reply:
@TheRiverwoodModder if you want to submit a PR restoring this, I believe I saw doodles approve so I'll merge it in; or maybe I can get to it first

Reply:
when it's merged the conflict will favour the other PR